modules/prebidServerBidAdapter/index.js

```diff
@@ -483,8 +483,21 @@ const OPEN_RTB_PROTOCOL = {
       tmax: _s2sConfig.timeout,
       imp: imps,
       test: getConfig('debug') ? 1 : 0,
+      // ext.prebid.targeting.includewinners is always true for openrtb endpoints
+      ext: {
+        prebid: {
+          targeting: {
+            includewinners: true
+          }
+        }
+      }
     };
 
+    // s2sConfig video.ext.prebid is passed through openrtb to PBS
+    if (typeof utils.deepAccess(_s2sConfig, 'video.ext.prebid') === 'object') {
+      request.ext.prebid = Object.assign(request.ext.prebid, utils.deepAccess(_s2sConfig, 'video.ext.prebid'));
```

Comment:
should `video.ext.prebid` be included for display (non-video) requests?

Reply:
If this is a multi format request, with multi imps with differing mediaTypes, then pbs needs the info to cache video bids and return banner bids fully.

So yes I think if it is specified it should be passed in.